src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition-duration: $transition;
```

Comment:
specify for which props you’re applying transition

Reply:
done